src/js/components/Filter.js

```diff
@@ -1,8 +1,7 @@
 // (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
 
 import React, { Component, PropTypes } from 'react';
-import pick from 'lodash/object/pick';
-import keys from 'lodash/object/keys';
+import { FormattedMessage } from 'react-intl';
```

Comment:
Sorry for missing this earlier. Should we be using Grommet FormattedMessage component instead?

Last I remember, if you are not using react-intl in the consuming application this would fail.

This is the Grommet component: https://github.com/grommet/grommet/blob/master/src/js/components/FormattedMessage.js

We don't know if consuming apps will be using react-intl or not.